apps/frontend/src/network/Network.tsx

```diff
@@ -65,7 +65,10 @@ const Network: React.FC<{}> = () => {
           filters={filters}
           searchQuery={searchQuery}
         >
-          <ErrorBoundary>
+          <ErrorBoundary
```

Comment:
Are we expecting the config for this component to vary a lot? Can these be default values?

Reply:
I was hoping we would put in more error boundaries with specific error messages where possible

Reply:
Yeah but as long as that doesn't happen? :sweat_smile: 
Or even then, if we have 5 with different custom descriptions and 5 with this generic one, seems good to have the component define a standard appearance with description and refresh link instead of duplicating the generic one

Reply:
I'm still unhappy about this :( maybe we can ask product if they intend to have this error message on all unexpected errors? I would kind of assume so though, because if we set a different error message then it's really an expected error isn't it

Reply:
Ive refactored it to have a more specific search results error.